src/pages/EditMonitor.vue

```diff
@@ -222,6 +222,32 @@
                                 {{ $t("Setup Notification") }}
                             </button>
 
+                            <!-- Proxies -->
+                            <h2 class="mt-5 mb-2">{{ $t("Proxies") }}</h2>
```

Comment:
Proxies section should be available for http(s) monitor type only?

Reply:
Correct, it is better to show only `v-if="monitor.type === 'http' || monitor.type === 'keyword' ">`